cookbook/docs/conf.py

```diff
@@ -305,6 +310,20 @@ def __call__(self, filename):
     # },
 }
 
+for i in range(len(sphinx_gallery_conf["examples_dirs"])):
```

Comment:
maybe I don't know what I'm talking about, but doesn't sphinx-gallery handle all of this?

Reply:
Nope, it doesn't. Sphinx gallery doesn't copy RST files (except README.rst). So I added a mechanism to always copy  RST files. The catch is to include them in the `toctree` in the concerned repo's README file.

Reply:
+1, please (you must approve for the merging to happen despite having two approvals as you've `requested for changes`). We need to merge this ASAP as the documentation isn't getting rendered properly.

Reply:
Merged it as the documentation isn't getting rendered properly. Let me know if there are any other changes required.